dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/dataset/tab/DatasetAnalysisTab.java

```diff
@@ -0,0 +1,154 @@
+package edu.harvard.iq.dataverse.dataset.tab;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.text.SimpleDateFormat;
+import java.util.List;
+import java.util.logging.Logger;
+
+import javax.ejb.EJB;
+import javax.faces.view.ViewScoped;
+import javax.inject.Inject;
+import javax.inject.Named;
+
+import org.omnifaces.util.Faces;
+
+import edu.harvard.iq.dataverse.DatasetFieldServiceBean;
+import edu.harvard.iq.dataverse.DataverseRequestServiceBean;
+import edu.harvard.iq.dataverse.PermissionsWrapper;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetField;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetFieldType;
+import edu.harvard.iq.dataverse.persistence.dataset.DatasetVersion;
+import edu.harvard.iq.dataverse.persistence.workflow.WorkflowArtifact;
+import edu.harvard.iq.dataverse.persistence.workflow.WorkflowExecution;
+import edu.harvard.iq.dataverse.settings.SettingsServiceBean;
+import edu.harvard.iq.dataverse.workflow.artifacts.WorkflowArtifactServiceBean;
+import edu.harvard.iq.dataverse.workflow.execution.WorkflowContext.TriggerType;
+import edu.harvard.iq.dataverse.workflow.execution.WorkflowExecutionServiceBean;
+
+@ViewScoped
+@Named("datasetAnalysisTab")
+public class DatasetAnalysisTab implements Serializable {
```

Comment:
I'm thinking if it should go in main code or mxrdr extension. We what do you think?